Exercises/checkNestingDepth.js

```diff
@@ -0,0 +1,4 @@
+const checkNestingDepth = (array = []) =>
+  JSON.stringify(array)
+    .split("")
+    .filter((character) => character === "[").length - 1;
```

Comment:
I think in this function checkNestingDepth should check the level of nesting of an array. Then, use that function to flatten the array into a single-dimensional array. but your function should return the last element in the array.
```js
checkNestingDepth([[[12, 3, 4, [1, 24, 4]]], [1, 23, 4], [[1, 5, 6]]]) //6
```